id/src/main/java/com/simprints/id/ScheduleBackgroundSyncUseCase.kt

```diff
@@ -21,6 +27,9 @@ class ScheduleBackgroundSyncUseCase @Inject constructor(
             imageUpSyncScheduler.scheduleImageUpSync()
             configManager.scheduleSyncConfiguration()
             authManager.scheduleSecurityStateCheck()
+            externalScope.launch {
```

Comment:
IIRC, the convention for the coroutines is to make things simple on the callers' side and ensure correct schedulers in the lowest layer possible (inside of the scheduler in this case).

It would also simplify the testing.

Reply:
Yep, that would be better. Will update.